### ISSUE TITLE
Weird line break in struct field with long attribute

### ISSUE THREAD

**Issue by sfackler** [rust-lang-nursery/rustfmt#2088]
The rustfmt used by the current rls-preview is inserting linebreaks in between the field name and type when there's a large attribute attached:

```rust
pub struct Endpoint {
    service_name: String,
    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
    ipv4:
        Option<Ipv4Addr>,
    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
    ipv6:
        Option<Ipv6Addr>,
    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
    port:
        Option<u16>,
}
```

**Comment by topecongiro** [comment]
This is fixed on the latest rustfmt-nightly (0.2.9). Unfortunately the current rls-preview uses the older version of rustfmt-nightly (0.2.7, AFAICT from the commit history of rls).

For now the choices are to build rls from source, or wait until the new release of rls gets published. I am sorry that I cannot help.

**Comment by sfackler** [comment]
Cool, no worries!
